### DISCUSSION TITLE
Feature Request - Make side by side diff windows resizable

### DISCUSSION THREAD

**Discussion by paulegradie** [community/community#7438] [Pull Requests]
Apologies if this has already been suggested!

So we all probably spend a lot of time performing code reviews at our respective companies. One thing I've noticed myself wishing for during code review on github is the ability to resize the side by side windows that I'm looking at to make minor (or major...) formatting adjustments. Typically this can be achieved by either looking at the modified file locally, or on github using the 'view file' feature (which will present the code at full width).

I think an excellent and valuable feature for the github UI (where I do most of my code review whilst simply reading the code) would be to enable users to resize the side by side windows. I'm thinking - make that divider down the middle draggable left and right so that lines that are a little too long no longer break in awkward spots.

That would drastically improve the readability of the code during review. 

I asked around my team where I work, and I feel like this could be a home run feature to implement.

For example (sorry about the censoring - just being overly cautious there):

![image](https://user-images.githubusercontent.com/30226456/140666331-c404bce3-fdfc-4337-ac3b-c4e7d6a6a595.png)

The red arrows show how I would drag the side by side diff left and right to resize each pane.

**Comment by jstutson-rel** [comment]
The left pain file tree also needs to be resizeable... I can't read most of my file names.

It doesn't even display a tool tip on hover.

**Comment by killergege** [comment]
I came here looking exactly for this. We are moving from Azure DevOps, and I find their PR review screen so much better.
This would be a good start.
